pkg/querier/queryrange/split_by_interval_test.go

```diff
@@ -349,3 +351,31 @@ func Test_evaluateAtModifier(t *testing.T) {
 		})
 	}
 }
+
+func Test_evaluateAtModifier_Error(t *testing.T) {
```

Comment:
My opinion is that we should merge this test into `Test_evaluateAtModifier`. We can extend `Test_evaluateAtModifier`'s parameter to have an optional `expectedErrorCode`, then go from there. 

The reason I think so is because this function seems quite duplicated to `Test_evaluateAtModifier`.

Reply:
Makes sense! I pushed a new commit with this change.